internal/handlers/commonparams/extract_params.go

```diff
@@ -88,7 +88,8 @@ func ExtractParams(doc *types.Document, command string, value any, l *zap.Logger
 		lookup := key
 
 		// If the key is the same as the command name, then it is a collection name.
-		if key == command {
+		// Depending on the driver, the key may be camel case or lower case for a collection name.
+		if strings.EqualFold(key, command) {
```

Comment:
Just compare `strings.ToLower` results if we must

Reply:
Let's do that. For now to ensure this test script passes `strings.ToLower` is needed on both `key` and `command`, https://github.com/FerretDB/FerretDB/pull/2460